frontend/src/lib/components/CommandPalette/commandLogic.ts

```diff
@@ -4,6 +4,7 @@ import { commandLogicType } from 'types/lib/components/CommandPalette/commandLog
 import Fuse from 'fuse.js'
 import { dashboardsModel } from '~/models/dashboardsModel'
 import { Parser } from 'expr-eval'
+import _ from 'lodash'
```

Comment:
Let's remove lodash from everywhere in the code. Most of what it provides can be done with ES6+ functions or small helpers in `lib/...`. Needing a [70kb](https://bundlephobia.com/result?p=lodash@4.17.20) (minified) dependency to get a few random elements from an array is not a good call.

Reply:
Closing since I mentioned it ~3x in the new batch of reviews